File: typescript/libs/workspace/domain/json/rxjs/system/src/lib/database/invoke/InvokeResult.ts
import { IInvokeResult, ISession } from '@allors/workspace/system';
import { InvokeResponse } from '@allors/protocol/json/system';
import { Result } from '../Result';

export class InvokeResult extends Result implements IInvokeResult {
  constructor(session: ISession, invokeResponse: InvokeResponse) {
    super(session, invokeResponse);
  }
}

File: dotnet/System/Workspace/Adapters/Allors.Workspace.Adapters.Remote/Database/Invoke/InvokeResult.cs
// <copyright file="RemoteInvokeResult.cs" company="Allors bv">
// Copyright (c) Allors bv. All rights reserved.
// Licensed under the LGPL license. See LICENSE file in the project root for full license information.
// </copyright>

namespace Allors.Workspace.Adapters.Remote
{
    using Allors.Protocol.Json.Api.Invoke;

    public class InvokeResult : Result, IInvokeResult
    {
        public InvokeResult(ISession session, InvokeResponse invokeResponse) : base(session, invokeResponse)
        {
        }
    }
}
